packages/flutter_tts/tizen/src/flutter_tts_tizen_plugin.cc

```diff
@@ -147,64 +87,46 @@ class FlutterTtsTizenPlugin : public flutter::Plugin {
       }
       result->Success(flutter::EncodableValue(0));
     } else if (method_name.compare("speak") == 0) {
-      if (speaking_) {
+      if (tts_->GetState() == TTS_STATE_PLAYING) {
         LOG_ERROR("[TTS] : You cannot speak again while speaking.");
         result->Success(flutter::EncodableValue(0));
         return;
       }
 
       if (std::holds_alternative<std::string>(arguments)) {
         std::string text = std::get<std::string>(arguments);
-        int utt_id;
-        ret = tts_add_text(tts_, text.c_str(), language_.c_str(), voice_type_,
-                           speed_, &utt_id);
-        if (ret != TTS_ERROR_NONE) {
-          LOG_ERROR("[TTS] tts_add_text failed: %s", get_error_message(ret));
+        if (!tts_->AddText(text)) {
+          LOG_ERROR("Invalid Arguments!");
```

Comment:
Failing `AddText()` doesn't indicate "Invalid Arguments".

Reply:
This log was added by my mistake. I will remove it